pennylane/math/grad.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
```

Comment:
We don't do Copyright 2018-2024 any more?

Reply:
Eh... no real consistent policy. Just something.